setup.cfg

```diff
@@ -6,11 +6,11 @@
 name = pyaleph
 description = Reference implementation of Aleph.im next generation network of decentralized big data applications.
 author = Moshe Malawach
-author-email = moshe.malawach@protonmail.com
+author_email = moshe.malawach@protonmail.com
 license = mit
 url = https://aleph.im
-long-description = file: README.rst
-long-description-content-type = text/markdown
+long_description = file: README.rst
```

Comment:
README.rst or README.md ?

Reply:
I would say .md indeed! I didn't check the validity of the fields.